lib/pages/child/plan_in_progress_page.dart

```diff
@@ -18,181 +31,232 @@ class _ChildPlanInProgressPageState extends State<ChildPlanInProgressPage> {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
+		return Scaffold(
 			body: Column(
 				crossAxisAlignment: CrossAxisAlignment.start,
 				children: [
-					AppHeader.widget(
-						title: '$_pageKey.header.title',
-						appHeaderWidget: ItemCard(
-							title: "Sprzątanie pokoju",
-							subtitle: "Co każdy poniedziałek, środę, czwartek i piątek",
-							isActive: true,
-							progressPercentage: 0.4,
-							activeProgressBarColor: AppColors.childActionColor,
-							chips:
-							<Widget>[
-								AttributeChip.withIcon(
-									icon: Icons.description,
-									color: AppColors.childBackgroundColor,
-									content: AppLocales.of(context).translate('page.childSection.panel.content.taskProgress', {'NUM_TASKS': 2, 'NUM_ALL_TASKS': 5})
-								)
-							],
-						),
-						helpPage: 'plan_info'
-					),
-					AppSegments(segments: _buildPanelSegments())
+					LoadableBlocBuilder<ChildTasksCubit>(
+						builder: (context, state) {
+							if(state is DataLoadInitial)
+								BlocProvider.of<ChildTasksCubit>(context).doLoadData();
+							else if (state is ChildTasksLoadSuccess)
+								return AppSegments(segments: _buildPanelSegments(state));
+							return Expanded(child: Center(child: CircularProgressIndicator()));
```

Comment:
The point of loadableBuilder is to handle the load & progressIndicator, you only need the AppSegment case here - look at other uses